product_docs/docs/biganimal/release/using_cluster/05c_upgrading_log_rep.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: Performing a major version upgrade of Postgres on BigAnimal
+navTitle: Upgrading Postgres major versions
+---
+
+## Using logical replication
+
+!!! Note
+This procedure does not work with distributed high-availability BigAnimal instances.
+!!!
+
+Logical replication offers a powerful method for upgrading major Postgres versions on BigAnimal instances, enabling a seamless transition with minimal downtime. 
+
+By replicating database changes in real-time from an older version to a newer one, this approach ensures data integrity and continuity. It's ideal for migrating data across different Postgres versions, providing a reliable upgrade path without sacrificing availability.
+
+!!! Important
+Depending on where your older and newer versioned BigAnimal instances are located, this procedure may accrue ingress and egress costs from your cloud service provider (CSP) for the migrated data. Please consult your CSP's pricing documentation to see how ingress and egress fees are calculated to determine any extra costs.
+!!!
+
+### Create a new BigAnimal instance
+
+Migrating between major versions first requires creating a new BigAnimal instance with the newer version of Postgres.
+
+The new instance must have sufficient storage to receive the data from the old instance, so ensure enough storage is provisioned when creating the new instance.
```

Comment:
To:  Ensure your target instance is provisioned with a storage size equal to or greater than your source instance.

Reply:
More clear and direct. Got it. 👍